docs/pages/reference/session-recording.mdx

```diff
@@ -0,0 +1,162 @@
+---
+title: Teleport Session Recording
+description: An overview of Teleport's session recording and its configuration
+---
+
+This page provides an overview of how Teleport records sessions and the various
+configuration options that apply to session recordings.
+
+Teleport supports two different recording configurations: synchronous and
+asynchronous recording. Additionally, Teleport administrators can configure
+where the recording takes place: at the node or at the proxy.
```

Comment:
We should make "node" and "proxy" are consistent with the rest of the docs (in terms of usage and capitalization), especially the Core Concepts page (https://goteleport.com/docs/core-concepts/). Does "proxy" refer to "Proxy Service" here? Also, does "node" refer to an SSH Service instance?

Reply:
Kind of. The use of these terms is somewhat intentional, as:

1. the config setting for these are literally `node` or `proxy`.
2. "proxy" typically means the "Proxy Service" but may also refer to the kube service or the windows desktop service, so the more generic proxy makes sense.
3. yes, 'node" refers to the SSH service